### ISSUE TITLE
Provide a `page.last` finder

### ISSUE THREAD

**Issue by deivid-rodriguez** [teamcapybara/capybara#1987]
I was writing some specs and since `page.first` worked, I assumed `page.last` would work too.

I think in some cases this simmetry is handy. For example, I was working on some JS that adds some tweaks to `first-child` and `last-child` elements of a container, so it would be nice to have this in order to test that.

I could work on a PR for this, just making sure there's interest first!

**Comment by abotalov** [comment]
https://github.com/teamcapybara/capybara/pull/1665
https://github.com/teamcapybara/capybara/issues/1733

**Comment by deivid-rodriguez** [comment]
I should've searched for some history, sorry.

However, #1665 was apparently rejected because of being "risky" since `all` didn't wait. But it waits in `capybara 3`, right? And #1733 is a related, but different, feature request.

This is how my specs currently look like

```ruby
it "can move questions up" do
  within "#survey-question-#{survey_question_2.id}-field" do
    click_button "Up"
  end

  first_question = page.first(".survey-question")

  expect(first_question).to have_field("survey[questions][][body_en]", with: "Second")
  expect(first_question).to have_button("Down")
  expect(first_question).to have_no_button("Up")
end

it "can move questions down" do
  within "#survey-question-#{survey_question_1.id}-field" do
    click_button "Down"
  end

  last_question = page.all(".survey-question").last

  expect(last_question).to have_field("survey[questions][][body_en]", with: "First")
  expect(last_question).to have_button("Up")
  expect(last_question).to have_no_button("Down")
end
``` 

I would like that they looked symmetric ... :smiley:

**Comment by twalpole** [comment]
@deivid-rodriguez I'm considering this, although I'm still leaning towards not including it and I'd love to just get rid of `first` if it wouldn't break so many peoples tests.  Newcomers to Capybara don't understand (or aren't willing to learn) the issues that `all`/`first` (and `last` if added) have and massively overuse them.  Yes the fact that `all` and `first` now wait by default will prevent some of the new user issues/confusion it won't fix the non-reloading issue.  

As an aside, if symmetry is your true goal you could just call `page.all(".survey-question").first` 😄

**Comment by deivid-rodriguez** [comment]
Well, the symmetry goal came after the surprise. My goal was to actually remove the surprise where `first` works but `last` does not.

Anyways, I definitely missed the reloading gotcha, I'm totally fine with first being dropped. Maybe you cndeprecate
